### PR TITLE
feat(TDI-41721) : support athenadb in jdbc component

### PR DESCRIPTION
**Pull request by akhabali**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-41721

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: services/components-maven-repo/pom.xml [hunk 1]
```diff
@@ -159,6 +159,20 @@
         </repository>
     </repositories>
 
+    <pluginRepositories>
+        <pluginRepository>
+            <id>sonatype-public-repository</id>
+            <url>https://oss.sonatype.org/content/groups/public</url>
```
Comment (gonchar-ivan) [inline services/components-maven-repo/pom.xml]: as I understand this repo is needed to get download-maven-plugin

File: services/components-maven-repo/pom.xml [hunk 2]
```diff
@@ -244,6 +258,26 @@
...
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.3.0</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC_2.0.7/AthenaJDBC41_2.0.7.jar</url>
```
Comment (gonchar-ivan) [inline services/components-maven-repo/pom.xml]: As I can see it is fat jar, which contains all dependencies. I just checked whether it doesn't contain transitive deps.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/9/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/9/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aakhabali%2FTDI-41721-athenadb)

File: services/components-maven-repo/pom.xml (hunk 1 above)
Comment (akhabali) [reply services/components-maven-repo/pom.xml]: yes

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/12/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/12/)

**Pull request merged**